faucet-service/.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
```

Comment:
Any reason to use circle instead of travis? I believe most of our projects are using Travis's free plan for open source project now.

Reply:
Circle is the configuration in the vapor template.